server/server.go

```diff
@@ -310,6 +467,10 @@ func (s *server) ErrorEncoder() kithttp.ErrorEncoder {
 	}
 }
 
+func (s *server) Logger() micrologger.Logger {
+	return s.logger
+}
```

Comment:
I plan to simplify the interface of the microkit server in the future. The current design is a little bit tedious.